stdlib/filepath/src/main/ballerina/src/filepath/file_path.bal

```diff
@@ -106,9 +121,12 @@ public function filename(string path) returns string|Error {
 # Returns the enclosing parent directory.
 # If the path is empty, parent returns ".".
 # The returned path does not end in a separator unless it is the root directory.
+# ```ballerina
+#  string|filepath:Error parentPath = filepath:parent("/A/B/C.txt");
+# ```
 #
-# + path - String value of file path.
-# + return - Path of parent folder or error occurred while getting parent directory
+# + path - String value of the file path
+# + return - Path of the parent folder or else an error if error occurred while getting the parent directory
```

Comment:
Inconsistent usage of terms. A directory is referred to as both, a "directory" and a "folder". Check all the places and fix. Better to stick with one term. Let's go with directory?

Reply:
The description for the error is not helpful. What sort of errors can you expect when using the function? Those need to be documented. Let's check and fix in all the relevant places.

Reply:
Corrected.